extensions/emmet/src/matchTag.ts

```diff
@@ -33,8 +44,11 @@ export function matchTag() {
 
 function getUpdatedSelections(editor: vscode.TextEditor, position: vscode.Position, rootNode: HtmlNode): vscode.Selection | undefined {
 	let currentNode = <HtmlNode>getNode(rootNode, position, true);
-	if (!currentNode) {
-		return;
+	if (!currentNode) { return; }
+
+	if (isTemplateScript(currentNode)) {
```

Comment:
Since `isTemplateScript` is called inside `getTemplateScriptNestedNode`, why call it specifically here?

We can simplify this with
```
currentNode = getTemplateScriptNestedNode(editor.document, currentNode, position) || currentNode;
```
Maybe rename `getTemplateScriptNestedNode` to `getNodeInsideScriptTemplateIfAny` ?